### ISSUE TITLE
How do you remove a failed installation?

### ISSUE THREAD

**Issue by Chris-M4gic** [neutrinolabs/pulseaudio-module-xrdp#86]
I installed this without problems on an Ubuntu 22.04 machine but when i was installing it on an identical machine I accidentally stoped  the  `scripts/install_pulseaudio_sources_apt_wrapper.sh` process while it was running and then when i try to run it again i get:

`** Remove old build root /var/lib/pa-build/dev before running this script`

 trying to delete the failed build with `sudo rm -rf /var/lib/pa-build/dev` returns:

rm: cannot remove '/var/lib/pa-build/dev/sys': Device or resource busy
rm: cannot remove '/var/lib/pa-build/dev/tmp': Device or resource busy
rm: cannot remove '/var/lib/pa-build/dev/home': Device or resource busy
rm: cannot remove '/var/lib/pa-build/dev/proc': Device or resource busy
rm: cannot remove '/var/lib/pa-build/dev/dev': Device or resource busy

When i try to see what process is using it  `lsof +D /var/lib/pa-build/dev`

It returns:

lsof: WARNING: can't stat() fuse.gvfsd-fuse file system /run/user/1000/gvfs
      Output information may be incomplete.
lsof: WARNING: can't stat() fuse.portal file system /run/user/1000/doc
      Output information may be incomplete.

If i `umount` those fuse mounts then i get nothing by  `lsof +D /var/lib/pa-build/dev`

I tried to kill the process id i get when i run `ps -ef | grep /var/lib/pa-build/dev` but it never gets terminated even though it returns a process terminated message. Rebooting, removing and purging everything i installed along pulseaudio doesn't seem to solve the issue.

Does anybody have any recommendations on how to get rid of that failed build?

Thanks in advance for any insights you may have.

**Comment by matt335672** [comment]
Hi @Chris-M4gic 

A reboot should definitely fix it. It certainly has for me in the past.

You say you've done that. What are the symptoms now?

Thanks.

**Comment by Chris-M4gic** [comment]
Thanks for your response @matt335672  It keeps coming back I rebooted multiple times and even disable xrdp and pulseaudio services but no matter what the same result kept coming back so we opted for no audio for now on that machine. It works just fine on the other one so it might be a unique case where something messed up the system files. I'll try to start in safemode and remove them. Feel free to close this out!
